### PR TITLE
Feature/upgrade widget common

### PR DESCRIPTION
**Pull request by rodrigopavezi**
@donnapep @stulees please review. cheers

### CODE REVIEW

File: test/integration/rss-horizontal-scroll.html [hunk 1]
```diff
@@ -80,10 +80,6 @@
       server.restore();
     });
 
-    test("should hide container", function() {
```
Comment (stulees) [inline test/integration/rss-horizontal-scroll.html]: @rodrigopavezi Is there a reason for removing this test?

Comment (rodrigopavezi) [inline test/integration/rss-horizontal-scroll.html]: @stulees the container is not hidden when the scroller is showing. Before widget common hide message was not setting display = block. It was setting the visible property. But if the container is not hidden so I guess we don't need to test for that. cheers

Comment (stulees) [inline test/integration/rss-horizontal-scroll.html]: @rodrigopavezi Okay thanks

**Comment by stulees** [comment]
@rodrigopavezi 👍

**Comment by rodrigopavezi** [comment]
@stulees actually, good that you raised that question. I took another look at that and it looks like the change to the message in common was making the container to become block after the init for horizontal scrolling set it to node. I fix that by setting the container to none after the message to be hide. Please review. cheers

**Comment by stulees** [comment]
@rodrigopavezi Nice catch 👍

**Pull request merged**